src/components/mount-in-app/mount-in-app.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text } from '@storybook/addon-knobs';
+import MountInApp from './mount-in-app';
+import notes from './notes.md';
+
+storiesOf('Mount In App', module)
+  .add('default', () => {
+    const children = text('children', 'Content to be mounted!');
+
+    return (
+      <div>
+        <div id='carbon-demo'>Some content to be replaced.</div>
+
+        <MountInApp
+          targetId='carbon-demo'
+        >
+          <div>{ children }</div>
+        </MountInApp>
+      </div>
+    );
+  }, {
+    info: {
+      text: `
+        # Mount In App component
```

Comment:
This markdown is a little bunched up and uneasy on the eye when rendered in the story. I'm not sure we want to have a header like that below the component itself, either. This applies to all the components, but it looks worst in this one. 

![image](https://user-images.githubusercontent.com/26925206/53878821-c32c0680-4004-11e9-8529-e8c74d152aab.png)

Reply:
Agree the markdown needs styling, had left as wasn't sure if we were going to be sorting out some universal styling for these further down the line, but happy to add now 👍 

I added all these description blocks off the back of the conversations in Slack yesterday, but looking now at what's actually on the existing demo site, I wonder if it might be better to just get rid of most of these bits, leaving just the component name / one-liner descriptions?

Reply:
I like the descriptive elements in the markup, and think they're an improvement on the current demo site. It's mainly the Header and the formatting.

Reply:
I've updated all of these to use JSX now, and have added some styles which give the whole thing a look consistent with the StoryBook defaults (the default `storybook-info` styles differed from these)